tests/lib/nested.sh

```diff
@@ -116,6 +117,23 @@ start_nested_core_vm(){
     fi
 }
 
+add_debug_output_on_nested_core_vm(){
+    cat <<EOF > local.conf
+[Unit]
+StartLimitInterval=0
+[Service]
+Environment=SNAPD_DEBUG_HTTP=7 SNAPD_DEBUG=1
+ExecStartPre=/bin/touch /dev/iio:device0
```

Comment:
Why do we need this?

Reply:
Apparently some tests use it.

I wonder if this should be done in the corresponding test rather than here where it raises eyebrows.

Reply:
IIRC, `/dev/iio*` is industrial I/O, it is a bit surprising to see it here.

Reply:
Well I copied that from the configuration we have for the snapd test suite because we could run any test on the nested vm but not sure if it is gonna happen. I could remove that and add it in case it is needed in the future.